product_restricted_type/readme/USAGE.md

```diff
@@ -0,0 +1 @@
+To use this module, you need to go to Inventory -> Configuration -> Product Categories and set a Restricted Product Type.
```

Comment:
Perhaps we can elaborate more on the features added here. Such as adding that `if the Restricted Product Type is added on a Product Category, if we then assign the Product Category to the Product Template, the type is automatically changed to the value from the Product Category.`